pkg/tanka/find.go

```diff
@@ -81,7 +81,7 @@ func findJsonnetFilesFromPaths(paths []string, opts FindOpts) ([]string, error)
 				jsonnetFiles, err := jsonnet.FindFiles(path, nil)
 				var mainFiles []string
 				for _, file := range jsonnetFiles {
-					if filepath.Base(file) == jpath.DefaultEntrypoint {
+					if filepath.Base(file) == jpath.DefaultEntrypoint || file == path {
```

Comment:
Would it perhaps make sense here to restrict that to only files with the suffix `.jsonnet`? For me, `*.libsonnet` files would not look like entrypoints to me. WDYT?